Epsilon.Abstractions/LearningDomainOutcomeResult.cs

```diff
@@ -2,8 +2,5 @@ namespace Epsilon.Abstractions;
 
 public record LearningDomainOutcomeResult(
     LearningDomainOutcome Outcome,
-    double Grade,
-    DateTime SubmittedAt,
-    string Assignment,
-    Uri AssignmentUrl
```

Comment:
Can you give reason for removing these?

Reply:
Because it's moved to `LearningDomainSubmission`  The parent object.

Reply:
Valid